src/MonthPickerInput.vue

```diff
@@ -1,78 +1,83 @@
 <template>
   <div
-    class="month-picker-input-container"
-    v-click-outside="hide"
+          class="month-picker-input-container"
+          v-click-outside="hide"
   >
     <input
-      class="month-picker-input"
-      type="text"
-      v-model="selectedDate"
-      @click="showMonthPicker()"
-      readonly
+            class="month-picker-input"
+            type="text"
+            v-model="selectedDate"
+            @click="showMonthPicker()"
+            readonly
     >
     <month-picker
-      v-show="monthPickerVisible"
-      @input="populateInput"
-      :default-year="defaultYear"
-      :default-month="defaultMonth"
-      :lang="lang"
-      :months="months"
-      :no-default="noDefault"
-      :show-year="showYear"
-      :clearable="clearable"
-      :variant="variant"
-      :editable-year="editableYear"
+            v-show="monthPickerVisible"
+            @input="populateInput"
+            :default-year="defaultYear"
+            :default-month="defaultMonth"
+            :lang="lang"
+            :months="months"
+            :no-default="noDefault"
+            :show-year="showYear"
+            :clearable="clearable"
+            :variant="variant"
+            :editable-year="editableYear"
     >
     </month-picker>
   </div>
 </template>
 
 <script>
-import Vue from 'vue'
-import MonthPicker from './MonthPicker.vue'
-import monthPicker from './month-picker'
+  import Vue from 'vue'
```

Comment:
No need to indent the script. I'm gonna add ESLint so it's easier to contribute.

Reply:
I leave the indentation where I think that is reasonable.